opal/static/js/test/patient_list.controller.test.js

```diff
@@ -207,8 +207,30 @@ describe('PatientListCtrl', function() {
 
     });
 
+  describe('refresh()', function() {
+      fit('should update .episodes with what the server thinks is the episode', function() {
```

Comment:
witness the fitness?

(just it)

Reply:
Good shout - and you got there first with the fix too :)